src/site/content/en/blog/squoosh-v2/index.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Announcing Squoosh v2
+subhead: New codecs support, updatd design, and CLI support!
+description: |
+  Major version change to Squoosh.app with new CLI feature.
+scheduled: true
+date: 2020-12-09
+hero: hero.jpg
+alt: A screeen capture of new Squoosh top page
+tags:
+  - blog
+  - tools
+  - javascript
+  - images
+---
+
+
+
+[Squoosh](https://squoosh.app) is an image compression app our team built and [debuted at Chrome Dev Summit 2018](https://youtu.be/ipNW6lJHVEs). We built it to make it easy to experiment with different image codecs, and to showcase the capabilities of the modern web.  
```

Comment:
Please break all lines at 100 characters. If you're using VS Code, [Rewrap](https://marketplace.visualstudio.com/items?itemName=stkb.rewrap) makes it pretty easy to do. The [Docs To X](https://gsuite.google.com/u/0/marketplace/app/docs_to_x/243920501110?hl=en&pann=docs_addon_widget) Google Docs Add-On also supports this.